Bio/Align/__init__.py

```diff
@@ -1620,8 +1570,8 @@ def map(self, alignment):
         >>> alignment1 = alignments1[0]
         >>> print(alignment1)
         AAAAAAAACCCCCCCAAAAAAAAAAAGGGGGGAAAAAAAA
-                |||||||-----------||||||        
-                CCCCCCC-----------GGGGGG        
```

Comment:
My editor was "helpful", but this kind of trailing whitespace is really annoying in doctests.